app/Http/Controllers/External/VatsimNet/ProcessVatsimNetWebhook.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers\External\VatsimNet;
+
+use App\Http\Controllers\BaseController;
+
+class ProcessVatsimNetWebhook extends BaseController
+{
+    public function __invoke()
+    {
+        if (request()->header('Authorization') !== config('services.vatsim-net.webhook.key')) {
+            return response()->json([
+                'status' => 'forbidden',
+            ], 403);
+        }
+
+        foreach (request()->json('actions') as $action) {
+            $class = config("services.vatsim-net.webhook.jobs.{$action['action']}");
```

Comment:
this is very laravel, but personally I'd have an explicit switch here with some logging for actions we don't understand

Reply:
Good shout.